integrations/pjrt/README.md

```diff
@@ -78,14 +77,7 @@ The plugin `openxla_pjrt_artifacts` is in the `ctstools` directory and
 performs additional manipulation of the environment in order to save
 compilation artifacts, reproducers, etc.
 
-## Contacts
-
-* [GitHub issues](https://github.com/openxla/openxla-pjrt-plugin/issues):
-  Feature requests, bugs, and other work tracking
-* [OpenXLA discord](https://discord.gg/pvuUmVQa): Daily development discussions
-  with the core team and collaborators
-
 ## License
 
-OpenXLA PJRT plugin is licensed under the terms of the Apache 2.0 License with
-LLVM Exceptions. See [LICENSE](LICENSE) for more information.
+IREE PJRT plugin is licensed under the terms of the Apache 2.0 License with
+LLVM Exceptions. See [LICENSE](../../LICENSE) for more information.
```

Comment:
Just redirect to the LICENSE in the root dir.

Reply:
I think this is accurate, thanks.

We could also mention that the API files are located at https://github.com/iree-org/iree/tree/main/integrations/pjrt/third_party/pjrt_c_api and those are also licensed under the Apache 2.0 License.

cc @marbre

Reply:
Good suggestion! Added.